src/navigation/NavBar.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Icon from '../media/Icon';
+import NavItem from './NavItem';
+
+export const NavBar = () => (
+	<NavItem
+		key={0}
```

Comment:
Confused about why this prop is needed here...

Reply:
I don't know. I'll investigate and remove it from the next PR since this is widely used in the next PR. This came with the copy and paste stuff.

Reply:
Investigated and decided to leave the key, it makes sense in the following PR. The key is because the items are part of an array.